public/sass/pages/_explore.scss

```diff
@@ -263,6 +263,9 @@
   display: flex;
   position: relative;
   align-items: flex-start;
+  @include media-breakpoint-down(sm) {
+    flex-wrap: wrap;
+  }
 
```

Comment:
class is only used in QueryRow so it doesn't affect anything else. When we hit 'sm' breakpoint, wrap Query Action Buttons if necessary.